### PR TITLE
[PAYLOAD][RNE] Add minimal RNE siren data

### PR DESCRIPTION
**Pull request by HAEKADI**
related to https://github.com/etalab/annuaire-entreprises-search-api/issues/332
depends on https://github.com/etalab/annuaire-entreprises-search-infra/pull/254

This pull request brings changes to the payload by:
- Introducing `date_mise_a_jour_rne`: the date when data was modified in RNE.
- Introducing `date_mise_a_jour_insee`: the date when data was modified in SIRENE (INSEE).
- Adjusting `date_mise_a_jour` from the INSEE date to the date when data was indexed in Elasticsearch.

### CODE REVIEW

File: aio/aio-proxy/aio_proxy/response/format_search_results.py [hunk 1]
```diff
@@ -100,7 +105,7 @@ def get_field(field, default=None):
     # Hide most fields if unité légale is non-diffusible
     is_non_diffusible = (
         True
-        if result_unite_legale.get("statut_diffusion_unite_legale") != "O"
+        if result_unite_legale.get("statut_diffusion_unite_legale") == "P"
```
Comment (XavierJp) [inline aio/aio-proxy/aio_proxy/response/format_search_results.py:108]: Why the change ? Is it for RNE only UL ?

**Review by XavierJp — APPROVED** [review]
Really nice maybe lets deploy this on tuesday to be able to test it ?

File: aio/aio-proxy/aio_proxy/response/format_search_results.py (hunk 1 above)
Comment (HAEKADI) [reply aio/aio-proxy/aio_proxy/response/format_search_results.py:108]: When `statut_diffusion_unite_legale` is None, and it almost always is for RNE siren numbers, the API hides the data otherwise thinking it is `non-diffusible`
